x/crosschain/types/message_abort_stuck_cctx_test.go

```diff
@@ -12,24 +14,18 @@ import (
 func TestMsgAbortStuckCCTX_ValidateBasic(t *testing.T) {
 	tests := []struct {
 		name string
-		msg  types.MsgAbortStuckCCTX
+		msg  *types.MsgAbortStuckCCTX
 		err  error
 	}{
 		{
 			name: "invalid address",
-			msg: types.MsgAbortStuckCCTX{
-				Creator:   "invalid_address",
-				CctxIndex: "cctx_index",
-			},
-			err: sdkerrors.ErrInvalidAddress,
+			msg:  types.NewMsgAbortStuckCCTX("invalid_address", "cctx_index"),
+			err:  sdkerrors.ErrInvalidAddress,
 		},
 		{
 			name: "valid",
-			msg: types.MsgAbortStuckCCTX{
-				Creator:   sample.AccAddress(),
-				CctxIndex: "cctx_index",
-			},
-			err: nil,
+			msg:  types.NewMsgAbortStuckCCTX(sample.AccAddress(), "cctx_index"),
+			err:  nil,
```

Comment:
Maybe lets a check in validate basic for the index and a corresponding test 
	if len(msg.CctxIndex) != 66 {
		return ErrInvalidCCTXIndex
	}
	
Alternatively, we can just create an issue for this and add this check in a separate PR . Might bee needed to add in multiple places